internal/records/events.go

```diff
@@ -156,7 +156,7 @@ func ReconcileEvents(ctx context.Context, now time.Time, ups map[string]Upness,
 
 	for key, up := range ups {
 		rec := events[key]
-		reconcileLog.Info("ReconcileEvents", "key", key, "expected", up.ExpectedCount, "ready", up.ReadyCount, "unknownCount", up.UnknownCount, "unknownThreshold", unknownThreshold)
+		reconcileLog.Info("ReconcileEvents", "key", key, "tpu_accelerator", up.Attrs.TPUAccelerator, "expected", up.ExpectedCount, "ready", up.ReadyCount, "unknownCount", up.UnknownCount, "unknownThreshold", unknownThreshold)
```

Comment:
is it necessary to include tpu_accelerator here?

Reply:
Not sure if we really need it here or just adding it to the event summary is suffice.

Reply:
I don't think it's needed here, you have a log entry like this for nodepools:
```
2025-10-20T19:04:42Z	INFO	aggregator.events	ReconcileEvents	{"type": "nodepools", "key": "js-7x-d016e", "expected": 1, "ready": 1, "unknownCount": 0, "unknownThreshold": 1}
```